tests/test_examples.py

```diff
@@ -33,7 +33,11 @@ def test_mnist():
 
 def test_pylearn2():
     filename = 'unittest_markov_chain'
-    pylearn2_test('train', filename, 0, 3, False)
+    try:
+        pylearn2_test('train', filename, 0, 3, False)
+    except OSError:
```

Comment:
If this test is broken for a good reason let's switch it off, at least temporarily because currently it is just a pain in the ass!